packages/bot-web-ui/src/components/dashboard/tutorial-tab/index.scss

```diff
@@ -13,7 +10,42 @@
 }
 
 .tutorials-wrap {
-    padding-bottom: 20rem;
+    .dc-dialog {
+        &__dialog {
+            width: 80vw;
+            height: 80vh;
+            max-width: unset;
+            max-height: unset;
```

Comment:
why set the max-width and height to unset? does this do anything in particular since you are already setting the width and height to 80

Reply:
this was done because the dialog which was being used was using core css max=height and max-width which was around 400px so i wanted to unset it. So it can take the height i specify.